pgmq-extension/sql/pgmq--1.4.4--1.5.0.sql

```diff
@@ -0,0 +1,84 @@
+DROP FUNCTION IF EXISTS pgmq.drop_queue(TEXT, BOOLEAN);
+CREATE OR REPLACE FUNCTION pgmq.drop_queue(queue_name TEXT, partitioned BOOLEAN)
```

Comment:
```suggestion
CREATE FUNCTION pgmq.drop_queue(queue_name TEXT, partitioned BOOLEAN)
```
As it was just deleted

Reply:
https://github.com/tembo-io/pgmq/blob/main/pgmq-extension%2Fsql%2Fpgmq--1.1.1--1.2.0.sql#L39
I did it this way to be consistent with other parts of the code. Should I still apply your changes?